packages/Icon/package.json

```diff
@@ -14,7 +14,7 @@
     "access": "public"
   },
   "scripts": {
-    "svgr": "npx @svgr/cli -d . src/svg --out-dir src/"
+    "svgr": "svgr -d . src/svg --out-dir src/"
```

Comment:
The reason of this change: `npx` use latest version of `@svgr/cli` from npmjs.com. It means that version lock (`"@svgr/cli": "^4.1.0"`) will be ignored. After this change Yarn will run `./node_modules/.bin/svgr`.

Reply:
good catch 💯